docs/config/README.md

```diff
@@ -1767,6 +1769,39 @@ By default the module will be shown if any of the following conditions are met:
 symbol = "∴ "
 ```
 
+## localip
+
+The `localip` module shows the IPv4 address of the primary network interface.
+
+### Options
+
+| Option     | Default                   | Description                                            |
+| ---------- | ------------------------- | ------------------------------------------------------ |
+| `ssh_only` | `true`                    | Only show IP address when connected to an SSH session. |
+| `format`   | `"[$localipv4]($style) "` | The format for the module.                             |
+| `style`    | `"bold yellow"`           | The style for the module.                              |
+| `disabled` | `false`                   | Disables the `localip` module.                         |
```

Comment:
I'm a little concerned about adding this module as enabled by default.
The prompt, when sshing, already has some default options that lengthen the prompt a fair bit: username and hostname. I worry that we may be making the prompt when sshing a little too long with the inclusion of local IP.
How would you feel about having this be disabled by default?

Reply:
Hi
I don't mind if this is disabled! I run my own config anyways :+1: